### ISSUE TITLE
Question regarding the average logits

### ISSUE THREAD

**Issue by sandylaker** [MaximeRobeyns/bayesian_lora#2]
Dear authors of Laplace-LoRA,

Thank you for your great work in this easy-to-use library. I have a minor question regarding the example usage.
Specifically, in 
https://github.com/MaximeRobeyns/bayesian_lora/blob/94e20a7c5b0fb7ac567608149d220d1563a8129c/examples/example_usage.py#L290

The logits are first normalized by softmax and then further averaged. However, this means that the resulted logits are all non-negative, right? However, logits can also be negative values.

It would be appreciated if you could provide some elaboration on the above questions. Thank you in advance.

**Comment by MaximeRobeyns** [comment]
Hi Yawei, thanks for finding this error. 

Indeed, the line you've highlighted calculates the averaged _probabilities_, when we mean to calculate the logits in the example. I've removed the softmax from the line in question in the example.

Thanks!
